src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -531,34 +527,6 @@ class spell_dk_death_grip_initial : public SpellScriptLoader
         }
 };
 
-// 48743 - Death Pact
-class spell_dk_death_pact : public SpellScriptLoader
-{
-    public:
-        spell_dk_death_pact() : SpellScriptLoader("spell_dk_death_pact") { }
-
-        class spell_dk_death_pact_AuraScript : public AuraScript
-        {
-            PrepareAuraScript(spell_dk_death_pact_AuraScript);
-
-            void HandleCalcAmount(AuraEffect const* /*aurEff*/, int32& amount, bool& /*canBeRecalculated*/)
-            {
-                if (Unit* caster = GetCaster())
-                    amount = int32(caster->CountPctFromMaxHealth(amount));
```

Comment:
This script needs updating, not removing.

New calculation is `amount = int32(caster->CountPctFromMaxHealth(GetSpellInfo()->GetEffect(EFFECT_2)->CalcValue(caster));` (with neccessary check in Validate that EFFECT_2 exists)